src/Authorization.AspNetCore/AuthorizationErrorMessageBuilder.cs

```diff
@@ -0,0 +1,107 @@
+#nullable enable
+
+using System.Linq;
+using System.Text;
+using GraphQL.Language.AST;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace GraphQL.Server.Authorization.AspNetCore
+{
+    public class AuthorizationErrorMessageBuilder : IAuthorizationErrorMessageBuilder
+    {
+        public string GenerateMessage(OperationType? operationType, AuthorizationFailure failure)
```

Comment:
```suggestion
        public string GenerateMessage(OperationType? operationType, AuthorizationResult result)
```
Also `AuthorizationResult.Failure` is nullable.

Reply:
why passing the whole AuthorizationResult?
GenearteMessage should only be called, if the result is not Suceeded. In that case, we can rely that failure is not null.
That's why failure is not nullable here.

Reply:
In order not to limit the context, for example, if one inherits from `AuthorizationResult`.